CONTRIBUTING.md

```diff
@@ -256,18 +294,28 @@ deactivate
 
 ### Document the change
 
-Charmcraft has frequent documentation updates to both its user guide and its developer
-material.
+Before you start documenting your changes, take a moment to familiarize yourself with
+the four categories of [Diátaxis](https://diataxis.fr), the framework that Charmcraft's
+documentation is built around.
+
+Most small changes call for updates to the existing pages that describe the current
+behavior. Look for how-to guides and references that mention the affected feature, and
+make any necessary changes.
+
+Major changes require new documentation describing the feature's usage and
+specifications. For example, if you implement a new CLI command, describe its usage in
+one of the how-to guides and create a new reference for its options and flags.
 
-Document feature changes and fixes in the relevant [release notes](docs/release-notes/).
+Ensure that feature changes and fixes are also documented in the relevant [release
+notes](docs/release-notes/).
```

Comment:
They aren't linked in the template. Did we forget there, or was this an innovation for Charmcraft?